runtime/src/actor_error.rs

```diff
@@ -14,41 +14,81 @@ pub struct ActorError {
     exit_code: ExitCode,
     /// Message for debugging purposes,
     msg: String,
+    /// Optional exit data
+    data: Option<IpldBlock>,
```

Comment:
Nit: since the data will be a real on-chain thing and the msg is just debug, move the data field up below the exit code so it's more prominent and grouped with the other persistent item.